Part 2 of the Claudine's article "Doing agile, using Scrum" including… (#222)
karakun/karakun.github.io: _posts/2023-09-22-agile-2.md

---
layout: post
title: 'Including values and principles in Scrum'
authors: ['claudine']
featuredImage: agile
excerpt: "In part 1 of this series of articles, we took a look back at the past and learned about the evolution of the agile idea behind the Manifesto for Agile Software Development. Let's see what we can do to bring the values and principles to our professional lives."
permalink: '/2023/09/22/agile-2.html'
categories: [Agile, Scrum]
header:
  text: Doing agile, using Scrum - an insight by practitioners
  image: post
---

> ## The incorporation of values and principles in Scrum

Reviewer: ```suggestion
```

> This is the second part of our series about scrum. In [part 1, "Doing agile, using Scrum"](/2023/05/05/agile-1.html), we have been looking back at the past and learned about the evolution of the "agile" idea behind the "Manifesto for Agile Software Development". Let's see what we can do to bring the values and principles of the Manifesto to action in our professional lives.

Reviewer: ```suggestion
This is the second part of our series about agile and scrum. In [part 1, "Doing agile, using Scrum"](/2023/05/05/agile-1.html), we have been looking back at the past and learned about the evolution of the "agile" idea behind the "Manifesto for Agile Software Development". Let's see what we can do to bring the values and principles of the Manifesto to action in our professional lives.
```

There are many discussions about the waste of time with all those events and Scrum not being able to come up with good results. But such conversations do not come up at a point where we check prerequisites:

* Is it the right approach to develop the product or for this project?
* Does the organization's culture allow an agile approach and self-managed teams?
* Do we know WHY we chose Scrum and what principles stand behind the framework?
* Do we know WHAT shall be achieved?

The early adopters of Scrum were at a different state of mind and focused on what does not work and changed it.

You won't succeed with Scrum if you assume to be successful by simply applying the Scrum Guide. There is no guarantee. Even if you consider the prerequisites, it will feel heavy and hurt sometimes. Which is a good thing - we mustn't ignore or be annoyed about. It allows improvement and, in some cases, even to choose an alternative to Scrum or other agile methods.

### So, in the end... or rather at the beginning...
Organizations should choose wisely what fits their needs - not what is "en vogue". So Scrum shouldn't be the default answer for each development initiative.

> In the upcoming article, we will have a deeper look into the realities of teams of projects - and what are some common pitfalls or misinterpretations. 

Reviewer: ```suggestion
In the upcoming article, we will have a deeper look into the realities of project teams and common pitfalls or misinterpretations. 
```
